# Human MDH2

# P40926

# phosphorylation of Y80 (pY56 in structure)

## Description

Analyzing the effects of the phosphorylation of TYR80, seen at residue 56, as well as the variant formed when replacing it with ASP56.

1.  image of the unmodified site ![The unmodified enzyme at TYR80](images/AF3_og.png)

2.  image of modification site ![The modified enzyme at TYR80, now phosphorylated](images/AF3_modification.png)

3.  image of the variant site ![The enzyme with ASP56 replacing TYR56](images/AF3_mimic.png)

4.  image of boltz-1 original at mimic site ![The enzyme modeled by boltz-1 at the variant site](images/Boltz_og.png)

5.  image of boltz-1 mimic at mimic site ![The variant enzyme modeled by boltz-1 at the variant site](images/Boltz_mimic.png)

6.  image of the AF3 mimic and AF3 original at active site ![The comparison of the mimic and original AF3 structures at the active site](images/AF3_og_mimic_sub.png)

7.  image of the modified and non-modified AF3 structures at active site ![The comparison of the modified and non-modified AF3 structures at the active site](images/og_mod_superimposed.png)

## Effect of the sequence variant and PTM on MDH dynamics

Part 3 from the Project 4 report

1.  Image of the site with the aligned PDB files (no solvent) ![image of original and mimic pdb files aligned in mol\*](images/Colab_aligned)

2.  Annotated Original RMSF plot showing differences between the simulations ![The original RMSF plot from the non-variant enzyme](images/Pr.2_RMSF.png)

3.  Annotated Variant RMSF plot showing differences between the simulations ![The RMSF plot from the variant enzyme](images/pr.4_RMSF.png)

4.  Annotated plots of original pKa for the key amino acids ![Graph showing the pKa values of the active site amino acids in the original structure, focusing on HIS176](images/Active_site_pKa_og.png)

5.  Annotated plots of variant pKa for the key amino acids ![box plot showing the pKa range for HIS176 in the variant enzyme](images/HIS176_pKa_bp.png)

6.  graph showing the pka trajectory in the original structure ![this graph shows the trajectory of pKa over time in the non-variant active site](images/Pr.2_pka_traj.png)

7.  graph showing the pKa trajectory in the variant structure ![this graph shows the pKa trajectory in active sight of the variant structure](images/Pr.4_pKa_traj.png)

8.  graph showing the RMSD data in the non-variant structure ![this graph shows the RMSD data over time in the non-variant structure](images/og_RMSD.png)

9.  graph showing the PMSD data for the variant structure ![this graph shows the RMSD data over time for the variant structure](images/mimic_RSMD.png)

Description of the data and changes

## Comparison of the mimic and the authentic PTM

![This image compares the mimic and modified structures at ARG86 in the active site](images/PTM_mim_ARG86.png)

Part 4 from the Project 4 report outline include images as needed

## Authors

Meredith Lawrence

## Date

05/07/2025

## License

Shield: [![CC BY-NC 4.0](https://img.shields.io/badge/License-CC%20BY--NC%204.0-lightgrey.svg)](https://creativecommons.org/licenses/by-nc/4.0/)

This work is licensed under a [Creative Commons Attribution-NonCommercial 4.0 International License](https://creativecommons.org/licenses/by-nc/4.0/).

[![CC BY-NC 4.0](https://licensebuttons.net/l/by-nc/4.0/88x31.png)](https://creativecommons.org/licenses/by-nc/4.0/)

## References

-   Christopher E. Berndsen, Jessica K. Bell; The structural biology and dynamics of malate dehydrogenases. Essays Biochem 3 October 2024; 68 (2): 57–72.![10.1042/EBC20230082](https://doi.org/10.1042/EBC20230082)

-   Cannon-Albright, L. A.; Stevens, J.; Teerlink, C. C.; Facelli, J. C.; Allen-Brady, K.; Welm, A. L. A Rare Variant in MDH2 (Rs111879470) Is Associated with Predisposition to Recurrent Breast Cancer in an Extended High-Risk Pedigree. Cancers 2023, 15 (24), 5851–5851.![10.3390/cancers15245851](https://www.mdpi.com/2072-6694/15/24/5851)

-   Priestley, J. R. C.; Pace, L. M.; Sen, K.; Aggarwal, A.; Alves, C. A. P. F.; Campbell, I. M.; Cuddapah, S. R.; Engelhardt, N. M.; Eskandar, M.; Jolín García, P. C.; Gropman, A.; Helbig, I.; Hong, X.; Gowda, V. K.; Lusk, L.; Trapane, P.; Srinivasan, V. M.; Suwannarat, P.; Ganetzky, R. D. Malate Dehydrogenase 2 Deficiency Is an Emerging Cause of Pediatric Epileptic Encephalopathy with a Recognizable Biochemical Signature. Molecular Genetics and Metabolism Reports 2022, 33, 100931.! [10.1016/j.ymgmr.2022.100931](https://doi.org/10.1016/j.ymgmr.2022.100931)

-   Laura de Lorenzo; Stack, Tyler M. M.; Fox, K.; Walstrom, K. Catalytic Mechanism and Kinetics of Malate Dehydrogenase. Essays in biochemistry 2024, 68 (2). ![10.1042/ebc20230086](https://doi.org/10.1042/ebc20230086)

-   Alabduladhem, T. O.; Bordoni, B. Physiology, Krebs Cycle. PubMed. <https://www.ncbi.nlm.nih.gov/books/NBK556032/>.

-   Abramson, J et al. Accurate structure prediction of biomolecular interactions with AlphaFold 3. Nature (2024).

-   Boltz-1, This Colab was adapted by Chris Berndsen at James Madison University (ver 2 2025) from a notebook written by Colabfold in 2024.

-   Mol\* Viewer. <https://molstar.org/viewer/> (accessed 2025-05-06).

-   MD Simulation, Colab adapted and developed by Angela Kayll, Angelina Sardelli, and Christopher Berndsen at James Madison University (ver 2 2025) with input and feedback from students in CHEM260 and members of the MCC community.

-   MD Analysis, Colab developed by Angela Kayll, Angelina Sardelli, and Christopher Berndsen at James Madison University (ver 2 2025) with input and feedback from students in CHEM260 and members of the MCC community.

-   UniProt. UniProt. <https://www.uniprot.org/uniprotkb/P40926/entry> (accessed 2025-05-06)

\*AlphaFold Filled - Model P40926. <https://alphafill.eu/model?id=P40926> (accessed 2025-05-06).
