---
editor_options: 
  markdown: 
    wrap: 72
---

# Human MDH1

# Uniprot ID: P40925

# Variation: Phosphorylation of S333

## Description

# Part 1 from Project 4 report outline (with citations as appropriate)

1.  Comparison of MDH1 Unmodified and Modified Structures ![Aligned
    Structures](images/AllAlignedStructures(2).png)

2.  Unmodified Modification Site ![Modification site (Ser332) of
    unmodified model of
    MDH1.](images/UnmodifiedModificationSite(AA333).png)

3.  Modification Site of Modified Model of MDH1 ![Modification site
    (Sep333) of modified model of
    MDH1.](images/ModifiedModificationSite.png)

4.  Modification Site of Mimic Model of MDH1 ![Modification site
    (Asp333) of mimic model of MDH1.](images/MimicModificationSite.png)

Uniprot lists a modified residue in the amino acid sequence at 333.6
Through exploring the literature, it is determined that this specific
modification site has not been described previously. MDH1 has a total of
334 amino acids with this modification of phosphorylation at the end of
the sequence; amino acid 333. More specifically, it is located at the
end of a helix, particularly far away and no interaction with the active
site at 187. The unmodified, modified, and mimic site were imported and
analyzed using Mol\* viewer. When looking at the modification site on
the unmodified model, the Ser333 interacts with Ser332, Glu328, Ala334,
and Phe329 (Figure 1). When looking at the mimic (variant) modification
site, Asp333 only interacts with Ser332 and Ala334 (Figure 2). And
finally, when looking at the modification site of the modified model,
Sep333 interacts with Lys103, Phe330, Ser332, and Ala334 (Figure 3).
Between the unmodified and modified model, there is only one amino acid
interaction difference: Glu328 for Lys103. Between the unmodified and
mimic models, the modification site has less amino acid interactions and
just includes interactions with its adjacent amino acids (Ser332 and
Alanine 334).

## Effect of the sequence variant and PTM on MDH dynamics

Part 3 from the Project 4 report

5.  Active Site - Unmodified Model ![Active Site of Unmodified
    MDH1](images/UnmodifiedActiveSite.png)

6.  Active Site - Mimic Model ![Active Site of Mimic
    MDH1](images/MimicActiveSite.png)

7.  Active Site - Modified Model ![Active Site of Modified
    MDH1](images/ModifiedActiveSite.png)

The unmodified active site is at His187 and interacts with Asn131,
Asn134, Asp159, Asn186, Ser188. When comparing it to the modified and
mimic versions, there are no differences within the active site, amino
acids the His187 interacts with, or weak interactions.

8.  RMSF Plot - Unmodified Model ![RMSF Plot of unmodified model of MDH1
    generated by Colab 2](images/UnmodifiedRMSF_plot(1).png)

9.  RMSF Plot - Mimic Model ![RMSF Plot of mimic model of MDH1 generated
    from Colab 2](images/RMSFPlotPicture.png)

Root mean square fluctuation (RMSF) plots were used to analyze the
flexibility and stability of MDH1 within the unmodified and mimic
models. The RMSF plot of the unmodified model was obtained from Project
3. The RMSF plot of the mimic model was generated using Colab 1 and 2.
When comparing the RMSF plots, areas of a higher RMSF value indicate
higher flexibility and areas of lower RMSF values indicate more rigid or
stable properties. Amino acids around 100 respectably had a peak of
around 3.1 in the unmodified graph (Figure 3). When looking at the same
peak, its highest point is around 2.1 in the mimic graph (Figure 4).
This may indicate changes in the substrate binding pockets, amino acids
92-98. This potentially means that the amino acids become more stable
and rigid in the mimic model. Additionally, a peak of around 1.5 at
amino acid \~330 was observed in the unmodified model (Figure 3). When
comparing this same peak at amino acid \~330, the peak is much larger at
around 2.4, indicating a very dynamic or flexible area (Figure 4). In
both Figure 3 and 4, there is no significant change around amino acid
187, MDH1’s active site.

10. pKa Box Plots ![pKa Box Plot for Unmodified
    Model](images/LABELLEDBOXPLOTFORREPORT.png) ![pKa Box Plot for Mimic
    Model](images/BoxPlotpKavalues.png)

When analyzing the pka values between the unmodified (Figure 5) and
mimic (Figure 6), there are not many differences. Amino acids 92, 98,
159, and 187 did not have significant changes. There were observable
small differences in the mimic model of the central pka values, but it
was not outside the standard deviation of these pka values in the
unmodified version. Additionally, there were no changes in the
protonated and deprotonated forms of these amino acids. At amino acid
162, there was a notable difference of a higher pka outside of the
standard deviation presented in the unmodified model. The unmodified
model’s amino acid 162 had a pka of 11.9 +/- 0.2. The mimic’s model
presented a pka of 13 +/- 0.2 in the same amino acid. However, there was
no change in the protonated forms between the unmodified and mimic
models.

## Authors

Mattie Herlinger

## Deposition Date

## License

Shield: [![CC BY-NC
4.0](https://img.shields.io/badge/License-CC%20BY--NC%204.0-lightgrey.svg)](https://creativecommons.org/licenses/by-nc/4.0/)

This work is licensed under a [Creative Commons
Attribution-NonCommercial 4.0 International
License](https://creativecommons.org/licenses/by-nc/4.0/).

[![CC BY-NC
4.0](https://licensebuttons.net/l/by-nc/4.0/88x31.png)](https://creativecommons.org/licenses/by-nc/4.0/)

## References

Bisong, E. Google Colaboratory. In Building Machine Learning and Deep
Learning Models on Google Cloud Platform; Apress: Berkeley, CA, 2019; pp
59–64. ![10.1007](https://doi.org/10.1007/978-1-4842-4470-8_7).

Haberman, A., Peterson, C.N. (2024). Genetics of MDH in humans. Essays
in biochemistry, 68(2), 107-119.
![10.1042](https://doi.org/10.1042/EBC20230078)

Lichtenstein, A.H. (2005). Fats and Oils: Phosphoserine. Encyclopedia of
Human Nutrition (Second Edition), 177-186.
![10.1016](https://doi.org/10.1016/B0-12-226694-3/00114-9).

Malate dehydrogenase, cytoplasmic. (2022). From AlphaFold protein
structure database. <https://alphafold.ebi.ac.uk/entry/P40925>

McCue, W., Finzel, W. (2021, 8 December) Structural Characterization of
the Human Cytosolic Malate Dehydrogenase. ACS Omega,7 (1),
207-214.![10.1021] DOI: 10.1021/acsomega.1c04385

The UniProt Consortium; Bateman, A.; Martin, M.-J.; Orchard, S.;
Magrane, M.; Adesina, A.; Ahmad, S.; Bowler-Barnett, E. H.; Bye-A-Jee,
H.; Carpentier, D.; Denny, P.; Fan, J.; Garmiri, P.; Gonzales, L. J. D.
C.; Hussein, A.; Ignatchenko, A.; Insana, G.; Ishtiaq, R.; Joshi, V.;
Jyothi, D.; Kandasaamy, S.; Lock, A.; Luciani, A.; Luo, J.; Lussi, Y.;
Marin, J. S. M.; Raposo, P.; Rice, D. L.; Santos, R.; Speretta, E.;
Stephenson, J.; Totoo, P.; Tyagi, N.; Urakova, N.; Vasudev, P.; Warner,
K.; Wijerathne, S.; Yu, C. W.-H.; Zaru, R.; Bridge, A. J.; Aimo, L.;
Argoud-Puy, G.; Auchincloss, A. H.; Axelsen, K. B.; Bansal, P.; Baratin,
D.; Batista Neto, T. M.; Blatter, M.-C.; Bolleman, J. T.; Boutet, E.;
Breuza, L.; Gil, B. C.; Casals-Casas, C.; Echioukh, K. C.; Coudert, E.;
Cuche, B.; De Castro, E.; Estreicher, A.; Famiglietti, M. L.; Feuermann,
M.; Gasteiger, E.; Gaudet, P.; Gehant, S.; Gerritsen, V.; Gos, A.;
Gruaz, N.; Hulo, C.; Hyka-Nouspikel, N.; Jungo, F.; Kerhornou, A.;
Mercier, P. L.; Lieberherr, D.; Masson, P.; Morgat, A.; Paesano, S.;
Pedruzzi, I.; Pilbout, S.; Pourcel, L.; Poux, S.; Pozzato, M.; Pruess,
M.; Redaschi, N.; Rivoire, C.; Sigrist, C. J. A.; Sonesson, K.;
Sundaram, S.; Sveshnikova, A.; Wu, C. H.; Arighi, C. N.; Chen, C.; Chen,
Y.; Huang, H.; Laiho, K.; Lehvaslaiho, M.; McGarvey, P.; Natale, D. A.;
Ross, K.; Vinayaka, C. R.; Wang, Y.; Zhang, J. UniProt: The Universal
Protein Knowledgebase in 2025. Nucleic Acids Research 2025, 53 (D1),
D609-D617. ![10.1093](https://doi.org/10.1093/nar/gkae1010).

Wohlwend, J.; Corso, G.; Passaro, S.; Getz, N.; Reveiz, M.; Leidal, K.;
Swiderski, W.; Atkinson, L.; Portnoi, T.; Chinn, I.; Silterra, J.;
Jaakkola, T.; Barzilay, R. Boltz-1 Democratizing Biomolecular
Interaction Modeling. November 20, 2024.
![10.1101](https://doi.org/10.1101/2024.11.19.624167).

Zhong Q, Xiao X, Qiu Y, Xu Z, Chen C, Chong B, Zhao X, Hai S, Li S, An
Z, Dai L. Protein posttranslational modifications in health and
diseases: Functions, regulatory mechanisms, and therapeutic
implications. MedComm (2020). 2023 May 2;4(3):e261.
![10.1002](https://doi.org/10.1002/mco2.261).
