# Human MDH2
# Q75MT9
# pS302


## The modification of Serine 302 in hMDH2 has not been previously researched. This residue was phosphorylated to become phosphoserine and a mimic variant was created. 


1. image of aligned unmodified and modified 
![PhospoS 302 and the unmodified structures were aligned with an RMSD of 1.41. The modified sequence is represented by green and orange, and the unmodified is represented by pink and blue. ](images/align_mod_unmod.png)
2. image of the unmodified site
![The unmodified site of serine 302 forms hydrogen bonds with surrounding serine, glycine, and glutamate.](images/unmod_site.png)

3. image of modification site
![Phosphoserine 302 forms hydrogen bonds with two additional amino acids including aspartic acid and asparagine. The modification site is near the active site His 176](images/mod_site.png)


## Effect of the sequence variant and PTM on MDH dynamics


1. Image of aligned PDB files (no solvent)
![The mimic variant post simulation and the modified protein were observed with a RMSD of 1.67. They had similar backbones and no changes near the active site or residue 302 compared to pre-simulation](images/pdb_align.png)![](images/pdb_align.png)

2. Image of the site with the aligned PDB files (no solvent)
![The area around the modified site is similar after the simulation was ran.The mimic variant has a builder side chain than before, becoming a better approximation for the modification.](images/site_pdb_align.png)

3. Annotated RMSF plot showing differences between the simulations
![The Root Square Mean Fluctuation was analyzed between the unmodified protein and the mimic. Key sites including the active site and amino acid 302 did not have significantly different RMSD values. There is a significant difference around amino acid 85, but this residue is not near either key sites. ](images/annotated_rmsf_plot.png)

4. Annotated plots of pKa for the key amino acids
![The modification did effect the pKa of the active site. The unmodified protein's active site has an average pKa of 5.06, and the modified protein is 6.98. This may limit the histidine's ability to donate protons in the modified protein.](images/pKa_over_traj.png)

5. If needed, show ligand bound images and how modification affects substrate binding

Description of the data and changes
1.Substrate binding in unmodified protein 
![The substrate S-malate binds to surrounding glycine, arginine, and the active site](images/s-malate_unmod.png)

2.Substrate binding in the modified protein and mimic variant
![The substrate S-malate binds to surrounding glycine, arginine, and the active site](images/s-malate_mod.png)


## Comparison of the mimic and the authentic PTM


![Substrate binding and the active site are similar in the mimic and the modified protein. ]
(images/mod_his.png)



## Authors

Summer Cocks

## Deposition Date

## License

Shield: [![CC BY-NC 4.0][cc-by-nc-shield]][cc-by-nc]

This work is licensed under a
[Creative Commons Attribution-NonCommercial 4.0 International License][cc-by-nc].

[![CC BY-NC 4.0][cc-by-nc-image]][cc-by-nc]

[cc-by-nc]: https://creativecommons.org/licenses/by-nc/4.0/
[cc-by-nc-image]: https://licensebuttons.net/l/by-nc/4.0/88x31.png
[cc-by-nc-shield]: https://img.shields.io/badge/License-CC%20BY--NC%204.0-lightgrey.svg


## References

