---
editor_options: 
  markdown: 
    wrap: 72
---

# Malate Dehydrogenase MDH2

# Uniprot ID: P40926

# Variation: phosphorylation of T171 and mimic of T171D

## Description

# Thr171 modification, whether by phosphorylation or the T171D mimic, affects the structure and function of MDH2. MDH is responsible for catalyzing the reversible oxidation of malate to oxaloacetate in the Krebs cycle, using NAD+/NADH as a cofactor.Proper positioning of amino acids within the protein is essential for catalytic efficiency. Especially proper positioning of residues involved in substrate binding or that act as a proton acceptor, such as Arg 152 or His 176, as seen in figure 1. That said, post-translational modifications (PTMs) and amino acid substitutions at or near the active site in MDH can interfere with important interactions and drastically alter its enzymatic activity.

1.  image of the unmodified site ![alt text](images/T147.png)

2.  image of modification site ![alt text](images/Modifiedandloop.png)

## Effect of the sequence variant and PTM on MDH dynamics

Part 3 from the Project 4 report

1.  [alt text](images/activesites.png)

2.  [alt text](images/activesites.png)

3.  Annotated RMSF plot showing differences between the simulations
    ![alt text](images/rmsf.png)

4.  Annotated plots of pKa for the key amino acids ![alt
    text](images/pka.png)

5.  If needed, show ligand bound images and how modification affects
    substrate binding

Description of the data and changes

## Comparison of the mimic and the authentic PTM

The shorter D side chain in the T171D mimic variation modifies local
bonding patterns without completely changing the loop structure. Even
though it maintains the interactions with Ile150 and Val151, the shorter
side chains cause weaker geometry. Furthermore, a new ionic connection
with Arg233 actually stabilizes the loop in a clearer, more confined
shape. A more noticeable loop displacement results with phosphorylation,
which adds a larger, highly charged phosphate group. This allows new
hydrogen bonds to form with residues like Cys251 and Glu250 (Figure 8).
This modification affects nearby structural components, slightly moving
areas that contain catalytic residues like Arg152 and Asp149. Thus, the
phosphorylated model exhibits more extensive structural alterations than
the mimic, highlighting the higher functional significance of genuine
phosphorylation.

### Colab notebook links

Copy_of_MD_simulation_Step1v2 Copy_of_MDanalysis_Step2

## Authors

Ashlyn Mazzocchi

## Deposition Date

## License

Shield: [![CC BY-NC
4.0](https://img.shields.io/badge/License-CC%20BY--NC%204.0-lightgrey.svg)](https://creativecommons.org/licenses/by-nc/4.0/)

This work is licensed under a [Creative Commons
Attribution-NonCommercial 4.0 International
License](https://creativecommons.org/licenses/by-nc/4.0/).

[![CC BY-NC
4.0](https://licensebuttons.net/l/by-nc/4.0/88x31.png)](https://creativecommons.org/licenses/by-nc/4.0/)

## References

-   Malate Dehydrogenase: A Story of Diverse Evolutionary Radiation.
    Essays in Biochemistry 2024. <https://doi.org/10.1042/ebc20230076>.

\*Betsy M. Martinez-Vaz; Howard, A.; Varuni K. Jamburuthugoda; Callahan,
K. Insights into the Regulation of Malate Dehydrogenase: Inhibitors,
Activators, and Allosteric Modulation by Small Molecules. Essays in
Biochemistry 2024. <https://doi.org/10.1042/ebc20230087>.

\*MDH2 malate dehydrogenase 2 [Homo sapiens (human)] - Gene - NCBI.
Nih.gov.
<https://www.ncbi.nlm.nih.gov/gene?Db=gene&Cmd=DetailsSearch&Term=4191>.

\*UniProt. www.uniprot.org.
<https://www.uniprot.org/uniprotkb/P40926/entry>.

\*ProtVar. EMBL-EBI ProtVar - Contextualising human missense variation.
Ebi.ac.uk.
<https://www.ebi.ac.uk/ProtVar/query?search=P40926+T171I&annotation=structural-row-1>
(accessed 2025-05-07).

\*AlphaFold Filled - Model P40926. Alphafill.eu.
<https://alphafill.eu/model?id=P40926&identity=25> (accessed
2025-05-07).

\*RMSD/RMSF Analysis \| BioChemCoRe 2018. ctlee.github.io.
<https://ctlee.github.io/BioChemCoRe-2018/rmsd-rmsf/>.

-   RMSD/RMSF Analysis \| BioChemCoRe 2018. ctlee.github.io.
    <https://ctlee.github.io/BioChemCoRe-2018/rmsd-rmsf/>.
