# Human MDH1

# P40925

# (S) 111p

## Description

# Part 1 from Project 4 report outline (with citations as appropriate)

1.  image of the unmodified site ![alt text](images/unmodified_active_site.png)

2.  image of modification site ![alt text](images/modified_active_site.png)

## Effect of the sequence variant and PTM on MDH dynamics

Part 3 from the Project 4 report

1.  Image of aligned PDB files (no solvent) ![alt text](images/all4alignedstructures.png)

2.  Image of the site with the aligned PDB files (no solvent) ![alt text](images/alignedsite.png)

3.  Annotated RMSF plot showing differences between the simulations ![alt text](images/mimic_RMSF.png) ![alt text](images/unmodified_rmsfplot.png) The RMSF plots showed differences in flexibility between the mimic and unmodified models. For example, the mimic modelâ€™s loop region near residue 111 showed more fluctuation. This suggests that the mimic is causing the loop to be more flexible, which could impact the active site function and overall enzyme function. Although weak interactions stayed the same, the increased flexibility could affect substrate binding or the positioning of active site residues.

4.  Annotated plots of pKa for the key amino acids ![alt text](images/unmodified_PKAtraj.png) ![alt text](images/mimic_PKAtraj.png)

The mimic fluctuated more than the unmodified model, suggesting that the surrounding charges of the mimic were less stable. This could be due to the introduction of the negative charge from changing the serine to the aspartate.

5.  If needed, show ligand bound images and how modification affects substrate binding

Description of the data and changes

## Comparison of the mimic and the authentic PTM

Part 4 from the Project 4 report outline include images as needed The mimic and modified models both had structures with low RMSD values, so the two structures were highly similar and there was no significant difference in their overall shapes. Therefore, the mimic provides a reasonable structure substitute for the modified version. The mimic and PTM retain the same hydrogen bonding at the active site residue HIS 186 and at nearby residues, including ASP 158, ASN 185, and SER 187. However, one hydrogen bond was present in the PTM model that was not in the mimic, suggesting a small difference in active site stabilization. ![alt text](images/rmsd_plot.png) ![alt text](images/mimic_active_site.png) ![alt text](images/modified_active_site.png)


## Authors

Syndey Brearley \## Deposition Date

## License

Shield: [![CC BY-NC 4.0](https://img.shields.io/badge/License-CC%20BY--NC%204.0-lightgrey.svg)](https://creativecommons.org/licenses/by-nc/4.0/)

This work is licensed under a [Creative Commons Attribution-NonCommercial 4.0 International License](https://creativecommons.org/licenses/by-nc/4.0/).

[![CC BY-NC 4.0](https://licensebuttons.net/l/by-nc/4.0/88x31.png)](https://creativecommons.org/licenses/by-nc/4.0/)

## References

-   Citation1 ![DOI](https://doi.org/10.1042/EBC20230079)

-   Citation2 ![DOI](DOI%20link)
