---
editor_options: 
  markdown: 
    wrap: 72
---

# Human malate dehydrogenase 1

# P40925

# 316p

## Description

# Malate dehydrogenase (MDH) is largely known as a crucial enzyme in metabolism and the citric acid cycle as it is responsible for converting malate to oxaloacetate and vice versa. MDH is important for more than just metabolism, it helps cells adapt to their environment through processes including the maintenance of redox balance, lipid synthesis, glutamine metabolism, and MDH supports the rapid increase of cells’ energetic and biosynthetic needs as they grow. The role of MDH in biosynthetic pathways is valuable as seen by the complexity of its’ regulatory mechanisms, for example post-translational modifications. The importance of metabolic regulation is well known, but when changes are made to an enzyme responsible for such regulation, risks for disease increase. Dysregulation of MDH in the body has relatively little literature investigating outcomes, although associations have linked increased MDH with multiple pathologies. The involvement of MDH dysregulation in breast cancer, prostate cancer, and pancreatic ductal adenocarcinomas suggest MDH plays a critical role for tumor cells adapting to metabolism. All in all, MDH plays a multifunctional role and has highly- specific regulatory mechanisms since its functions are of high importance to mammal survival (Parente et al. & Provost et al.). This report highlights how alteration of a single amino acid can impact the structure of the enzyme. By modifying the amino acid threonine in the 316th base pair position through phosphorylation and mimicking the enzyme by replacing threonine with aspartic acid in the 316th base pair position, this report analyzes how alterations in biochemical interactions may lead to changes in MDH’s highly regulated function.

1.  image of the unmodified site ![Unmodified site of residue 316 in
    MDH1](images/unmodified_site.png)

2.  image of modification site ![Modified site of residue 316 in
    MDH1](images/modified_site.png)

## Effect of the sequence variant and PTM on MDH dynamics

Part 3 from the Project 4 report

1.  Image of aligned PDB files (no solvent) ![Aligned unmodified,
    modified, and PTM mimic of MDH1](images/md_align.png)

2.  Image of the site with the aligned PDB files (no solvent) ![Aligned
    unmodified, modified, and PTM mimic site of interest in
    MDH1](images/md_site.png)

3.  Annotated RMSF plot showing differences between the simulations
    ![Figure 10. RMSF plot for PTM mimic model. Figure 11. Unmodified
    RMSF plot from project 2](images/RMSF_plots.png)

4.  Annotated plots of pKa for the key amino acids ![pKa values of
    residues 316, 314, and 318 in the PTM mimic model of
    MDH1.](images/pKa_plot.png)

5.  If needed, show ligand bound images and how modification affects
    substrate binding

Description of the data and changes

## Comparison of the mimic and the authentic PTM

Part 4 from the Project 4 report outline include images as needed

### Colab notebook links

Provide file names of completed colab notebooks

## Authors

Contributors names

## Deposition Date

## License

Shield: [![CC BY-NC
4.0](https://img.shields.io/badge/License-CC%20BY--NC%204.0-lightgrey.svg)](https://creativecommons.org/licenses/by-nc/4.0/)

This work is licensed under a [Creative Commons
Attribution-NonCommercial 4.0 International
License](https://creativecommons.org/licenses/by-nc/4.0/).

[![CC BY-NC
4.0](https://licensebuttons.net/l/by-nc/4.0/88x31.png)](https://creativecommons.org/licenses/by-nc/4.0/)

## References

-   Citation1 ![DOI](DOI%20link)

-   Citation2 ![DOI](DOI%20link)
