---
editor_options: 
  markdown: 
    wrap: 72
---

# Homo sapiens MDH2

# P40926

# Acetylation of K91 (K67 in structure)

## Description

The modification site in human malate dehydrogenase 2 at amino acid 91
has been researched before. In human MDH2 the amino acid at 91 is
normally lysine but from a particular sequence from UnitProt, the lysine
is at position 67.

# Part 1 from Project 4 report outline (with citations as appropriate)

1.  image of the unmodified site ![Aligned of MDH2, MDH2 with acetylK91
    (purple), and K91Q
    (pink)](images/05062025_AF3_unmodified_PTM_mimic_aligned.png)

2.  image of modification site ![Modification site alignment within
    MDH2](images/05052025_AF3_aligned_unmodified_variant_PTM.png)

## Effect of the sequence variant and PTM on MDH dynamics

Part 3 from the Project 4 report

1.  Image of aligned PDB files (no solvent) ![Aligned unmodified shown
    in red and variant in
    green](images/05062025_colab_aligned_mimic_unmodified_pdb.png)

2.  Image of the site with the aligned PDB files (no solvent) ![Variant
    Modification MDH2 Site,
    GLN67](images/05062025_colab_mimic_modsite.png)

3.  Annotated RMSF plot showing differences between the simulations
    ![Variant RMSF plot](images/05042025_rmsf_plot_mimic.png)

4.  Annotated plots of pKa for the key amino acids ![Variant
    pKa](images/05042025_pka_HIS176_mimic.png)

5.  If needed, show ligand bound images and how modification affects
    substrate binding

After the simulation, the overall structure is similar although slight
changes can have effects on the function of MDH2.

## Comparison of the mimic and the authentic PTM

![Aligned modified shown in green and variant shown in
purple](images/05062025_AF3_aligned_mimic_PTM.png)

The glutamine and acetylated lysine both still form hydrogen bonds like
to the unmodified lysine. While both of these alterations maintain
similar hydrogen bonding, their effects on pKa suggest that they do not
fully replicate the function of MDH2.

### Colab notebook links

05012025_Colab_MD_simulation_Step1v2.ipynb
05012025_Colab_of_MDanalysis_Step2.ipynb

## Authors

Shana Love

## Deposition Date

## License

Shield: [![CC BY-NC
4.0](https://img.shields.io/badge/License-CC%20BY--NC%204.0-lightgrey.svg)](https://creativecommons.org/licenses/by-nc/4.0/)

This work is licensed under a [Creative Commons
Attribution-NonCommercial 4.0 International
License](https://creativecommons.org/licenses/by-nc/4.0/).

[![CC BY-NC
4.0](https://licensebuttons.net/l/by-nc/4.0/88x31.png)](https://creativecommons.org/licenses/by-nc/4.0/)

## References

-   Lisa M. Baird, Christopher E. Berndsen, Jonathan D. Monroe; Malate
    dehydrogenase in plants: evolution, structure, and a myriad of
    functions. Essays Biochem 3 October 2024; 68 (2):
    221–233.[10,1042](https://doi.org/10.1042/EBC20230089)

-   Takahashi-Íñiguez, T.; Aburto-Rodríguez, N.; Vilchis-González, A.
    L.; Flores, M. E. Function, Kinetic Properties, Crystallization, and
    Regulation of Microbial Malate Dehydrogenase. J Zhejiang Univ Sci B
    2016, 17 (4),
    247–261.[10.1631](https://doi.org/10.1631/jzus.B1500219)

-   Venkat, S.; Gregory, C.; Sturges, J.; Gan, Q.; Fan, C. Studying the
    Lysine Acetylation of Malate Dehydrogenase. J Mol Biol 2017, 429
    (9), 1396–1405.[10.1016](https://doi.org/10.1016/j.jmb.2017.03.027)

-   Pogostin, B. H.; Malmendal, A.; Londergan, C. H.; Åkerfeldt, K. S.
    pKa Determination of a Histidine Residue in a Short Peptide Using
    Raman Spectroscopy. Molecules 2019, 24 (3),
    405.[10.3390](https://doi.org/10.3390/molecules24030405)
