---
editor_options: 
  markdown: 
    wrap: 72
---

# Humans Malate Dehydrogenase 2

# P40926

# Phosphorylation

## Description

Malate dehydrogenase plays a critical role in the human metabolism, specifically the Kreb’s cycle. The two isoforms of MDH can differ in location sites within the cell, MDH1 (in the cytosol) and MDH2 (in the mitochondrial matrix). MDH1 has the function of assisting in the malate-aspartate shuttle by sustaining the redox balance of the cellular components within the cell. The malate-aspartate shuttle aids in malate moving into the mitochondrial to convert into oxaloacetate 2. The other isoform, MDH2, catalyzes the reversible reaction of malate oxidizing into oxaloacetate with the cofactor of NAD+, thus aiding in the formation of ATP to create energy for the cell. This function of MDH2 is crucial for the management of NAD+/NADH between the cytosol and mitochondria 3. Throughout the body, differing components are reliant on MDH to participate in energy production. MDH deficiencies and abnormalities can lead to various diseases and harm the body in many ways such as neurological diseases like epilepsy, neurological development delays, intellectual disabilities, and abnormalities in the brain 1. On the other hand, overexpression of MDH2 has been associated with many different cancers growing and resistance to cancer fighting drugs. It has been determined that this overexpression changes the metabolism of cancer cell known as the Warburg effect as the metabolic process changes to aerobic glycolysis 4. Other effects of excessive MDH2 are increasing cancer cell survival through redox balance of oxygen and aiding cancer cells in acting like stem cells 5. Through these findings it has become evident that MDH2 inhibition would be a vastly important treatment for cancer patients.

1.  image of the unmodified site ![Alignment and visual representation
    of unmodified site of MDH2 in Mol\* from information gathered from
    AlphaFold3](images/Unmodified.png)

2.  image of modification site ![Alignment and visual representation of
    modified site of MDH2 in Mol\* from information gathered from
    AlphaFold3](images/Modified.png)

3.  image of mimic site from AlphaFold3 ![Alignment and visual
    representation of mimic site of MDH2 in Mol\* from information
    gathered from AlphaFold3](images/MimicAF3.png)

4.  image of mimic site from Boltz-1 ![Alignment and visual
    representation of mimic site of MDH2 in Mol\* from information
    gathered from Boltz-1](images/MimicB1.png)

## Effect of the sequence variant and PTM on MDH dynamics

1.  Image of aligned PDB files (no solvent) ![Alignment and visual
    representation of whole enzyme of MDH2 in Mol\* from information
    gathered from AlphaFold3 and Boltz-1](images/Enzymealigned.png)

2.  Image of the site with the aligned PDB files (no solvent)
    ![Alignment and visual representation of site in MDH2 in Mol\* from
    information gathered from AlphaFold3 and
    Boltz-1](images/Alignedsite.png)

3.  Annotated unmodified RMSF plot showing differences between the
    simulations ![RMSF plot of the unmodified
    sequence](images/UnmodifiedRMSF.png)

4.  Annotated mimic RMSF plot showing differences between the
    simulations ![RMSF plot of the mimic sequence from
    AlphaFold3](images/mimicRMSF.png)

5.  Annotated plots of pKa for the active site ![pKA plot of active site
    His176 comparing the unmodified (blue)and the mimic
    (red)](images/pKaHis176.png)

6.  Annotated plots of pKa for the residue amino acid Arg86 near the
    active site ![pKA plot of residue amino acid Arg86 comparing the
    unmodified (blue)and the mimic (red)](images/pKaArg86.png)

7.  Annotated plots of pKa for the residue amino acid Arg152 near the
    active site ![pKA plot of residue amino acid Arg152 comparing the
    unmodified (blue)and the mimic (red)](images/pKaArg152.png)

Description of the data and changes

## The change from the original sequence’s Ser237 to the mimic’s sequence Asp237 was due to the change of S to D in the sequence. This resulted in a hydrophobic contact bond to Val 248 compared to the hydrogen bond to Glu 250. The change in bonds leads to a loss of stabilization and an unfavorable interaction between Asp 237 that is polar, acidic and hydrophobic amino acid to a Val248 that is nonpolar, neutral, and hydrophobic. This interaction can result in lack of stability, enzyme efficiency/specificity, and alteration of the active site geometry. The mimic structure differences between AlphaFold3 and Boltz-1 concludes that the certainty of the bond coming off the chain cannot be explicitly established. AlphaFold3’s depiction of the bond to Val248 can be due to possible Van Der Waal forces and simply the proximity. Without the bond coming off the side chain, the stability decreases allowing for the side chain and the surrounding amino acids to be susceptible to changes in structure that can ultimately impact function. The modified sequence that phosphorylated the serine amino acid did not cause any changes for interactions with surrounding amino acids and type of bonds for those interactions. The phosphorylation increased acidity and changed the side chain to a deprotonated state with a negative charge. This difference can impact folding, increase steric hinderance, attract positively charged amino acids, and repel negatively charged amino acids. Due to its location not being located near or at His176, Asn118, and Met227, the effects of this change wouldn’t impact the mechanism of malate and overall function but instead could impact surrounding interactions.

## Authors

Isabelle Rijksen

## May 7, 2025

## License

Shield: [![CC BY-NC
4.0](https://img.shields.io/badge/License-CC%20BY--NC%204.0-lightgrey.svg)](https://creativecommons.org/licenses/by-nc/4.0/)

This work is licensed under a [Creative Commons
Attribution-NonCommercial 4.0 International
License](https://creativecommons.org/licenses/by-nc/4.0/).

[![CC BY-NC
4.0](https://licensebuttons.net/l/by-nc/4.0/88x31.png)](https://creativecommons.org/licenses/by-nc/4.0/)

## References

-   Dasgupta, S., Ewing-Graham, P. C., Groenendijk, F. H., Stam, O.,
    Biermann, K. E., Doukas, M., Dubbink, H. J., van Velthuysen, M. F.,
    Dinjens, W. N. M., and Van Bockstal, M. R. (2020) Granular dot-like
    staining with MLH1 immunohistochemistry is a clone-dependent
    artefact. Pathology - Research and Practice 216, 152581.
    [j.prp.2019.152581](https://doi.org/10.1016/j.prp.2019.152581)

-   MDH1 Malate Dehydrogenase 1 [Homo Sapiens (Human)] - Gene - NCBI.
    National Center for Biotechnology Information, U.S. National Library
    of Medicine, April 7, 2025.
    (<https://www.ncbi.nlm.nih.gov/gene?Db=gene&Cmd=DetailsSearch&Term=4190>)

-   MDH2 Malate Dehydrogenase 2 [Homo Sapiens (Human)] - Gene - NCBI.
    National Center for Biotechnology Information, U.S. National Library
    of Medicine, April 7, 2025.
    (<https://www.ncbi.nlm.nih.gov/gene?Db=gene&Cmd=DetailsSearch&Term=4191>)

-   Roth, K. G.; Mambetsariev, I.; Kulkarni, P.; Salgia, R. The
    Mitochondrion as an Emerging Therapeutic Target in Cancer. Trends
    Mol. Med. 2020, 26 (1), 119–134.
    [j.molmed.2019.06.009](https://doi.org/10.1016/j.molmed.2019.06.009)

-   Santoro, A.; Campolo, M.; Liu, C.; Sesaki, H.; Meli, R.; Liu, Z.-W.;
    Kim, J. D.; Diano, S. DRP1 Suppresses Leptin and Glucose Sensing of
    POMC Neurons. Cell Metab. 2017, 25, 647–660.
    [10.1016/j.cmet.2017.01.003](https://doi.org/10.1016/j.cmet.2017.01.003)

-   Wang, H.; Yang, X.; Wang, M.; Hu, M.; Xu, X.; Yan, A.; Hao, Q.; Li,
    H.; Sun, H. Atomic Differentiation of Silver Binding Preference in
    Protein Targets: Escherichia coli Malate Dehydrogenase as a
    Paradigm. Chem. Sci. 2020, 11, 11714–11719.
    [10.1039/D0SC04151C](https://doi.org/10.1039/D0SC04151C)
